README.md

```diff
@@ -41,7 +41,7 @@ There are some required configurations and setup that must be completed before r
     Access to an AWS account is required.  Please contact `help@excella.com` to gain access to the ExcellaLabs account.  Regardless of the account, an `access_key` and `secret_key` are required for programatic access to the environment.   
 1. AWS CLI install and Configuration  
     Once access to an AWS account is available, as well as an `access_key` and `secret_key`, the [AWS CLI Tool](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) must be installed, and configured by running `aws configure`  
-1. Configure the AWS CLI when using MFA
+1. Configure the AWS CLI when using MFA [why aren't we just having them use `awsmfa`?]
     Special consideration is required when your AWS account is protected with a MFA device.  To configure the CLI, a `token` is required, which will provide temporary credentials.
```

Comment:
for now you could put (or use awsmfa, <link>)

Reply:
and put a link

Reply:
Added steps for using awsmfa.